.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: php
 
 php:
-  - 7.0
```

Comment:
Why remove these lines? In compsoer.json there is still a line that requires at least PHP 7.0, so I'd say we should also keep on testing on hese versions.

Reply:
Because PHPUnit 8 or greater needs PHP 7.2 at least. A better option is using [Phan](https://github.com/phan/phan). I can implement it if you want to.